pkg/raft/testdata/async_storage_writes.txt

```diff
@@ -505,8 +505,6 @@ process-ready 1 2 3
   HardState Term:1 Vote:1 Commit:13 Lead:1 LeadEpoch:1
   Entries:
   1/15 EntryNormal "prop_4"
-  CommittedEntries:
```

Comment:
Why is this test changing?

Reply:
I still need to audit it and confirm, but I think: there can only be one `MsgStorageApply` in flight now, so some commits migrated between `Ready`s.